docs/extensions.rst

```diff
@@ -133,3 +133,38 @@ TLS
 ---
 
 See :doc:`specs/tls`.
+
```

Comment:
Moving @synodriver's comment to its own thread:

> > I think that multiple http.response.trailers messages is an unnecessary complication
> 
> Actually when you are using [h11](https://pypi.org/project/h11/), there is only one chance to send trailer: at the end of response body. So it's ok for servers which only support http/1.1. But when it comes to http/2, things changed. Mutiple trailers is allowed, and, to be honest, I spend quite a lot time thinking trailers under http/2, and found that may need some [tricky](https://github.com/nonebot/nonecorn/issues/11). I have no idea if it's allowed.

I'm not sure how this has implications in the spec, but I'd like to highlight this comment, as it may get lost.